records/benchmarks/0.17.3--after.txt

```diff
@@ -0,0 +1,467 @@
+        "sampleVariance": 4.494494852795147e-12
+      },
+      "completed": true,
+      "percentSlower": 77.1
+    },
+    {
+      "ms": 0.008708222303498964,
+      "name": "rot13()",
+      "ops": 114834,
+      "margin": 1.46,
+      "options": {
+        "delay": 0.005,
+        "initCount": 1,
+        "minTime": 0.05,
+        "maxTime": 5,
+        "minSamples": 5
+      },
+      "samples": 101,
+      "promise": false,
+      "details": {
+        "min": 0.000008547856068376068,
+        "max": 0.000015105795249549008,
+        "mean": 0.000008708204301893064,
+        "median": 0.000008610405982905984,
+        "standardDeviation": 6.504187458084054e-7,
+        "marginOfError": 1.2684940489266376e-7,
+        "relativeMarginOfError": 1.4566654673580424,
+        "standardErrorOfMean": 6.471908412891009e-8,
+        "sampleVariance": 4.2304454489897904e-13
+      },
+      "completed": true,
+      "percentSlower": 0
+    },
+    {
+      "ms": 0.8424599831508003,
+      "name": "copycat.sentence()",
+      "ops": 1187,
+      "margin": 1.4,
+      "options": {
+        "delay": 0.005,
+        "initCount": 1,
+        "minTime": 0.05,
+        "maxTime": 5,
+        "minSamples": 5
+      },
+      "samples": 87,
+      "promise": false,
+      "details": {
+        "min": 0.0007431153333333334,
+        "max": 0.0009723327205882353,
+        "mean": 0.0008423057986108361,
+        "median": 0.0008279535072463768,
+        "standardDeviation": 0.000056118934373892705,
+        "marginOfError": 0.000011792499344862033,
+        "relativeMarginOfError": 1.4000259008439317,
+        "standardErrorOfMean": 0.000006016581298398996,
+        "sampleVariance": 3.149334795261276e-9
+      },
+      "completed": true,
+      "percentSlower": 98.97
+    }
+  ],
+  "fastest": {
+    "name": "rot13()",
+    "index": 1
+  },
+  "slowest": {
+    "name": "copycat.sentence()",
+    "index": 2
+  }
+}
+Running "1000 words" suite...
+{
+  "name": "1000 words",
+  "date": "2023-09-28T20:44:23.417Z",
+  "results": [
+    {
+      "ms": 0.4930966469428008,
+      "name": "copycat.scramble()",
+      "ops": 2028,
+      "margin": 0.16,
+      "options": {
+        "delay": 0.005,
+        "initCount": 1,
+        "minTime": 0.05,
+        "maxTime": 5,
+        "minSamples": 5
+      },
+      "samples": 99,
+      "promise": false,
+      "details": {
+        "min": 0.00048720226213592235,
+        "max": 0.000507408572815534,
+        "mean": 0.0004930614399833094,
+        "median": 0.0004920271067961165,
+        "standardDeviation": 0.0000040034587512890795,
+        "marginOfError": 7.886309776276861e-7,
+        "relativeMarginOfError": 0.15994578234598553,
+        "standardErrorOfMean": 4.02362743687595e-7,
+        "sampleVariance": 1.6027681973273117e-11
+      },
+      "completed": true,
+      "percentSlower": 79.35
+    },
+    {
+      "ms": 0.10182262498727217,
+      "name": "rot13()",
```

Comment:
I added `rot13()` in the comparison here because I doubt it is possible to scramble faster than string rotation. So its something that we're unlikely to beat or get close to, but the closer we get to it the better. `0.49ms` compared to `0.1ms` didn't seem too bad though.